apps/studio/components/interfaces/Auth/Policies/Policies.tsx

```diff
@@ -57,8 +60,9 @@ const Policies = ({
     },
   })
   const { mutate: deleteDatabasePolicy } = useDatabasePolicyDeleteMutation({
-    onSuccess: () => {
+    onSuccess: async () => {
       toast.success('Successfully deleted policy!')
+      await queryClient.invalidateQueries(lintKeys.lint(ref))
```

Comment:
Should we move this to `useDatabasePolicyDeleteMutation` definition? It should always invalidate the lint rules, right?